pkg/tnf/identifier/identifiers.go

```diff
@@ -199,3 +202,21 @@ var VersionIdentifier = Identifier{
 	URL:             versionIdentifierURL,
 	SemanticVersion: versionOne,
 }
+
+// ServiceAccountIdentifier is the Identifier used to represent the generic serviceAccount test.
+var ServiceAccountIdentifier = Identifier{
+	URL:             serviceAccountIdentifierURL,
+	SemanticVersion: versionOne,
+}
+
+// RoleBindingIdentifier is the Identifier used to represent the generic roleBinding test.
+var RoleBindingIdentifier = Identifier{
+	URL:             roleBindingIdentifierURL,
+	SemanticVersion: versionOne,
+}
+
+// ClusterRoleBindingIdentifier is the Identifier used to represent the generic clusterRoleBinding test.
+var ClusterRoleBindingIdentifier = Identifier{
+	URL:             clusterRoleBindingIdentifierURL,
+	SemanticVersion: versionOne,
+}
```

Comment:
Please add appropriate entries to the Catalog map here:
https://github.com/test-network-function/test-network-function/blob/main/pkg/tnf/identifier/identifiers.go#L70